core/genxyz.f

```diff
@@ -1496,7 +1496,7 @@ subroutine mesh_metrics
       do ie = 1,nelt
          ddmin = min(ddmin,dxmin_e(ie)) 
          ddmax = max(ddmax,dxmax_e(ie))
-         dtmp  = vlsum(JACM1(1,1,1,ie),nxyz)**1./3
+         dtmp  = vlsum(bm1(1,1,1,ie),nxyz)**1./ldim
```

Comment:
Why was `bm1` changed to `JACM1`?

Reply:
Jacobian determinant is a scale from changing the variables. It doesn't have unit.
The sum of bm1 is volume, and the cubic root of it is length scale.

Take one element = [-1,1]^3 as an example,
jacm1 are all ones so
```
vlsum(JACM1(1,1,1,ie),nxyz)**1./3 = lx1 / N
```
On the other hand, 
```
vlsum(bm1(1,1,1,ie),nxyz)**1./3   = 2 / N
```
which is what we expect for [-1,1] in N intervals.

We didn't print this number out anyway. I just found this inconsistency while checking the code...

Reply:
If it's not used we should simply remove it.